services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall_id,
+        movie=movie_id
```

Comment:
In the `create_movie_session` function, `cinema_hall` and `movie` should be assigned instances of `CinemaHall` and `Movie`, respectively, not their IDs. You need to retrieve the instances using `CinemaHall.objects.get(id=cinema_hall_id)` and `Movie.objects.get(id=movie_id)` before assigning them.